user-registration-app/src/app/views/live-workshop/live-workshop.component.ts

```diff
@@ -7,25 +7,49 @@ import { Component, OnInit, Input } from '@angular/core';
   styleUrls: ['./live-workshop.component.css'],
 })
 export class LiveWorkshopComponent implements OnInit {
+  private _date: Date;
+  private _startTime: Date;
+  private _endTime: Date;
+
   @Input() title: string;
   @Input() description: string;
-  @Input() date: string;
-  @Input() time: string;
   @Input() link: string;
   @Input() skills: string;
   @Input() downloads: string[];
-  @Input() presenter1img: string;
-  @Input() presenter1Name: string;
-  @Input() presenter2img: string;
-  @Input() presenter2Name: string;
-  @Input() presenter3img: string;
-  @Input() presenter3Name: string;
+  @Input() presenters: string[];
   @Input() collapseID: string;
+  // @Input() presenter1img: string;
```

Comment:
Small thing, any reason these are commented instead of just removed?

Reply:
We still are working out what to do to handle these presenter images. Do you think we should host them all locally with the site according to presenter name?